### ISSUE TITLE
Hack biogenerator for new options

### ISSUE THREAD

**Issue by Dagdammit** [yogstation13/yogstation#1714]
[Note]: # (State what the issue is from a "whats wrong" prospective. Issue reports should clearly allow maintainers to understand whats wrong and how to test/reproduce if that is not obvious. Avoid ambiguity. Start your issue report below both of these lines (or remove them))

[Admins]: # (If you are reporting a bug that occured AFTER you used varedit/admin buttons to alter an object out of normal operating conditions, please verify that you can re-create the bug without the varedit usage/admin buttons before reporting the issue.)

**Comment by Dagdammit** [comment]
Other things you COULD also add to biogenerator, though I'm on the fence about all of these & leaning towards "no" (even if they all cost like 20k biomass):

-Combat Gloves
-Galoshes
-30u bottles of reagents (maybe with some sketchy thing about their appearance?) found in weeds that can overtake trays. (Believe those include charcoal, morphine, sulphuric acid, nutriment, amatoxin and mushroom hallucinogen)

**Comment by Cruix** [comment]
No traitor items, maybe straight jackets. Maybe even move mutagen under the hacked options, since experienced botanists hack the seed vendor all the time anyway.

**Comment by Dagdammit** [comment]
Mutagen under hacked options would be neat, though my heart goes out a
little to newbie botanists who'll dearly wish they'd learned about that
option sooner. Straitjacket's a smart thought too.

Smuggler's Satchels and Military Belts seemed okay to me because they're
just fabric/leather in a different configuration, not something that had to
be developed in some advanced syndicate lab. That, and they're both
underused items that could be put to interesting uses- and if sec kicks
your ass for having contraband then that's a consequence you shoulda been
prepared for. (t-ray scans reveal satchels, right?)

**Comment by matskuman5** [comment]
Make it so when it's emagged, you can shove a dead corpse into it and turn them into biomass.

**Comment by Dagdammit** [comment]
I'd be down if it made a bloody mess everywhere like with chef.

**Comment by Rafael450** [comment]
dont move the mutagen to the hacked biogenerator, how about perma, we wont be able to mutate our plants?

**Comment by Dagdammit** [comment]
Rafael's got a good point.

**Comment by Cruix** [comment]
Even if anyone used perma for anything other than storing braindeads, perma does not need to be able to mutate deathnettles.

**Comment by Rafael450** [comment]
Thats not the point, we need to give the prisoner a chance to escape without killing the warden, for example, i use Ambriosa Deus to space walk.

**Comment by Dagdammit** [comment]
I've got a theory about combining teslium and water to EMP doors, unsure if
it's viable.

But yeah, botany-based perma escape is tricky, dangerous, takes time and
ingenuity and skill, can be nipped in bud but only if sec actually pays
attention. Locking away mutagen would ruin that. It needs to be a default
option for the perma biogenerator, even if hydro one still requires hacking.

**Comment by Rafael450** [comment]
i agree with Dagdammit, the perma biogenerator should be hacked by default, AND JUST IF SEC GIVE YOU A PLANT BAG, since its hard to keep doing in manual once you get a good yeild and production.

**Comment by Cruix** [comment]
You can mutate things with left-4-zed, it just takes longer and has downsides. Any halfway competent botanist can get essentially infinite mutagen from the biogenerator in ten minutes, which translates to infinite mutation chances and maxed potency.

**Comment by Dagdammit** [comment]
Not with the prison setup, limited seed selection and 4 trays counts for
alot. And you gotta load plants by hand, IIRC.
